fix revision display

Bartzi opened the pull request:
fixes #620

janno42 commented on _1327/documents/tests.py:
```diff
@@ -153,6 +153,50 @@ def test_revert_to_different_url(self):
 		self.assertIn(reverse('versions', args=[old_url]), response.body.decode('utf-8'))
 
+	def test_version_creation(self):
+		Document.objects.get().delete()
```
shouldn't this be `Document.objects.all().delete()`?

janno42 commented on _1327/documents/tests.py:
```diff
@@ -153,6 +153,50 @@ def test_revert_to_different_url(self):
+
+			self.assertEqual(Document.objects.count(), expected_count)
+			document = Document.objects.get(url_title=url_title)
+			response = self.app.get(reverse('minutes:edit', args=[document.url_title]), user=self.user)
```
why `minutes:edit`?

Bartzi commented:
I've addressed your comments.
Regarding the server:
- we already determined that the most current version of a document is not identical to the current version, which it should be
- so we should look for all documents where the last revision is not identical to the current text and then add a new revision with the current version, that should fix it. But I recommend doing that on a test system

janno42 commented on _1327/documents/utils.py:
```diff
@@ -49,12 +49,13 @@ def handle_edit(request, document, formset=None, initial=None, creation_group=No
-				document.save_formset(formset)
-				revisions.set_user(request.user)
-				revisions.set_comment(cleaned_data['comment'])
+				with revisions.create_revision():
```
can you please add a comment what this does and why `revisions.create_revision()` has to be where it is?

janno42 replied:
meaning why it needed to be moved from the line above
